google/cloud/examples/gcs2cbt.cc

```diff
@@ -223,7 +223,14 @@ int main(int argc, char* argv[]) try {
   }
   std::cout << " DONE" << std::endl;
 
-  gcs::Client client(gcs::ClientOptions().set_project_id(project_id));
+  google::cloud::StatusOr<gcs::ClientOptions> status_or_opts =
```

Comment:
Generally we have been naming these variables `client_options` or `opts`, just like we would name a `unique_ptr<ClientOptions> client_options` and not `client_options_unique_ptr` or something.